setup.cfg

```
@@ -31,7 +31,7 @@ dev =
     attrs>=19.3.0
     black==22.3.0
     bluesky>=1.11.0
-    caproto[standard] >=0.4.2rc1
+    caproto[standard] >=0.4.2rc1,<1.2.0
```

```suggestion
    caproto[standard] >=0.4.2rc1,!=1.2.0
```

excluding a specific version to avoid a bug is safer as if we forget to remove it won't prevent fixed versions from working (and we may want to avoid 1.12.0 forever).

---

PR with this change here so we can merge sooner: https://github.com/bluesky/ophyd/pull/1241